examples/http-compression/http_compression.bal

```diff
@@ -0,0 +1,98 @@
+import ballerina/http;
+import ballerina/log;
+
+endpoint http:Listener listenerEndpoint {
+    port: 9090
+};
+
+// Since compression behaviour of the service is set as COMPRESSION_AUTO, entity body compression is done according
+// to the scheme indicated in `Accept-Encoding` request header. When the header is not present or the
```

Comment:
Shall we reword this to "Compression is not performed when the header is not present or the header value is "identity"."